pkg/controllers/proxy/network_services_controller.go

```diff
@@ -1470,48 +1538,88 @@ func (nsc *NetworkServicesController) buildEndpointsInfo() endpointsInfoMap {
 	return endpointsMap
 }
 
+type IptablesHandler struct {
```

Comment:
I would love some propositions about how to name that struct. `IptablesHandler` doesn't seem right, but that's the only name I came with.

So, in general, that structure is supposed to keep all things needed to use iptables for **a specific IP family**. One instance of that struct contains iptables handler and node IP, CIDR in version 4. The other instance contains ip6tables handler and node IP, CIDR in version 6.

The purpose of that structure is to have a slice of them, so the common code using iptables can do something like:

```
for _, iptablesHandler := range iptablesHandlers {
    iptablesHandler.iptablesCmdHandler.AppendUnique("nat", "POSTROUTING", "-j", "SNAT", "--to-source", iptablesHandler.nodeIP)
}
```

I find it better than doing

```go
if nsc.enableIPv4 {
    iptablesHandler.iptablesCmdHandler.AppendUnique("nat", "POSTROUTING", "-j", "SNAT", "--to-source", nsc.nodeIPv4)
}
if nsc.enableIPv6 {
    iptablesHandler.iptablesCmdHandler.AppendUnique("nat", "POSTROUTING", "-j", "SNAT", "--to-source", nsc.nodeIPv6)
}
```

Maybe something like `IptablesFamilyInfo`? `IptablesFamilyHandler`?

Honestly, I probably need to use kind of abstraction in the other modules as well, to avoid the `if.enableIPv6` copy&paste craziness, but this module was the biggest one, so I felt motivated to start from it.